ui/src/app/pages/admin/admin-users.tsx

```diff
@@ -28,60 +34,58 @@ const styles = reactStyles({
 const LockoutButton: React.FunctionComponent<{disabled: boolean,
   profileDisabled: boolean, onClick: Function}> =
   ({disabled, profileDisabled, onClick}) => {
-    return <Button type='secondaryLight' onClick={onClick} disabled={disabled}>
+    // We reduce the button height so it fits better within a table row.
+    return <Button type='secondaryLight' style={{height: '40px'}} onClick={onClick} disabled={disabled}>
       {disabled ? <Spinner size={25}/> : (profileDisabled ? 'Enable' : 'Disable')}
     </Button>;
   };
 
+interface Props {
+  profileState: {
+    profile: Profile, reload: Function, updateCache: Function
+  };
+}
+
+interface State {
+  contentLoaded: boolean;
```

Comment:
nit: what's the difference between `isLoaded` and `contentLoaded`? Maybe standardizing on one would help.